byzcoin/bcadmin/clicontracts/value.go

```diff
@@ -0,0 +1,177 @@
+package clicontracts
```

Comment:
How about if this directory is simply called `contracts`. Then there would be byzcoin/contracts/*.go and byzcoin/bcadmin/contracts/*.go for the server and client sides.

Reply:
That's what I originally did but it seems that this is not allowed. I get the following error:
```
contracts redeclared as imported package name previous declaration at...
```

Reply:
The fix for this, if you wanted, would have been to rename the import of the byzcoin/contracts package, like this:

```
package conrtacts
import (
  bcContracts "go.dedis.ch/cothority/v3/byzcoin/contracts"
)
```

But calling this directory clicontracts is fine too.